src/Core/Settings.h

```diff
@@ -574,6 +574,8 @@ class IColumn;
     M(DateTimeInputFormat, date_time_input_format, FormatSettings::DateTimeInputFormat::Basic, "Method to read DateTime from text input formats. Possible values: 'basic' and 'best_effort'.", 0) \
     M(DateTimeOutputFormat, date_time_output_format, FormatSettings::DateTimeOutputFormat::Simple, "Method to write DateTime to text output. Possible values: 'simple', 'iso', 'unix_timestamp'.", 0) \
     \
+    M(BoolFormat, bool_format, FormatSettings::BoolFormat::true_false_lower_case, "Text to represent bool value. Possible values: 'true_false', 'True_False', 'T_F', 'Yes_No', 'Y_N', 'On_Off'.", 0) \
```

Comment:
I think we can support fully custom true/false representation for TSV/CSV, not only these possible values. It can be done almost the same as custom null representation for these formats (see settings `format_tsv_custom_null_representation/format_csv_custom_null_representation`). What do you think?

Reply:
I think this implementation is great！